### PR TITLE
Create board class

### PR DESCRIPTION
**Pull request by robertguturing**
## What is the pull request for?
* I wanted to create a board class for the tic tac toe game.

## Is there anything in particular you want review for?
* Help me I don't know how to write tests :(

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Board
+  attr_accessor :spaces
```
Comment (memcmahon) [inline lib/board.rb]: I wonder if this needs to be an accessor?  Would a reader be sufficient?

File: lib/board.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  def initialize
+    @spaces = ['', '', '', '', '', '', '', '', '']
+  end
+
+  def spaces
```
Comment (memcmahon) [inline lib/board.rb]: Oh, maybe this is the reader - we could make this an attr_reader method.

**Review by memcmahon — COMMENTED** [review]
Great start! let me know what you think of my notes :)

File: lib/board.rb (hunk 1 above)
Comment (robertguturing) [reply lib/board.rb]: That makes sense, i'll change it!

File: lib/board.rb (hunk 2 above)
Comment (robertguturing) [reply lib/board.rb]: totally! you are so smart!

**Comment by memcmahon** [comment]
Do you want to schedule some time to pair on tests for this class?

**Comment by memcmahon** [comment]
Looks good - lets merge!

**Pull request merged**